File: C++/A1/047.cpp
#include <iostream>
using namespace std;
int main(){
    int kap,clock,time;
    cin>>kap>>clock;
    time = kap*clock;
    if(time==0){
        cout<<"No teaching";
    }else{
        if((time/60)==0){
            cout<<time%60<<" minutes";
        }else if((time%60)==0){
            cout<<time/60<<" hours";
        }else{
            cout<<time/60<<" hours "<<time%60<<" minutes";
        }
    }
}

File: Python/A1/047.py
N = int(input())
A = int(input())
total_time = N * A
if total_time == 0:
    print("No teaching")
else:
    if (total_time//60)==0:
        print(f"{total_time%60} minutes")
    elif (total_time%60)==0:
        print(f"{total_time//60} hours")
    else:
        print(f"{total_time//60} hours {total_time%60} minutes")